pgx/src/memcxt.rs

```diff
@@ -239,6 +239,16 @@ impl PgMemoryContexts {
         }
     }
 
+    /// Returns parent memory context if any
+    pub fn parent(&self) -> Option<PgMemoryContexts> {
+        let parent = unsafe { pg_sys::MemoryContextGetParent(self.value()) };
```

Comment:
```suggestion
        // SAFETY: We do this instead of simply plucking the .parent field ourselves
        // mostly to let Postgres check the context validity if --enable-cassert is on
        let parent = unsafe { pg_sys::MemoryContextGetParent(self.value()) };
```